obstacles:
  - name: "Obstacle1"
    position: [0.25, 0.5, 0.6]
    velocity: [0.0, 0.0, 0.0]
    radius: 0.05
    is_dynamic: true
    angular_speed: 0.5
  - name: "Obstacle2"
    position: [-0.13, -0.52, 0.5]
    velocity: [0.0, 0.0, 0.0]
    radius: 0.05
    is_dynamic: true
    angular_speed: 0.5
  - name: "Obstacle3"
    position: [0.25, 0.3, 0.6]
    velocity: [0.0, 0.0, 0.0]
    radius: 0.06
    is_dynamic: true
    angular_speed: 0.5
  - name: "Obstacle4"
    position: [0.30, 0.3, 0.6]
    velocity: [0.0, 0.0, 0.0]
    radius: 0.06
    is_dynamic: true
    angular_speed: 0.5
  


